### PR TITLE
WIP on docs

### PR DESCRIPTION
**Pull request by blinkov**

### CODE REVIEW

**Comment by blinkov** [comment]
@zhang2014 @sundy-li this touches few zh articles too

**Comment by sundy-li** [comment]
ok

File: docs/zh/interfaces/third-party/integrations.md [hunk 1]
```diff
@@ -3,12 +3,17 @@
 !!! warning "放弃"
```
Comment (sundy-li) [inline docs/zh/interfaces/third-party/integrations.md]: !!! warning "声明"

File: docs/zh/interfaces/third-party/gui.md [hunk 2]
```diff
@@ -36,5 +38,29 @@ ClickHouse Web 界面 [Tabix](https://github.com/tabixio/tabix).
 - 集群管理
 - 监控副本情况以及 Kafka 引擎表
 
+## 广告
```
Comment (sundy-li) [inline docs/zh/interfaces/third-party/gui.md]: 广告  ->   商业

**Comment by blinkov** [comment]
@sundy-li thanks

**Pull request merged**